src/simba/dashboard/zonal_manager/branch_revenue.html

```
@@ -4,286 +4,278 @@
 
-          <th scope="col">
-            <!-- Sort Dropdown -->
-            <div class="hs-dropdown relative inline-flex w-full cursor-pointer">
-              <button id="hs-pro-eptsas" type="button" class="px-5 py-2.5 whitespace-nowrap text-start w-full flex items-center gap-x-1 text-sm font-normal text-stone-500 focus:outline-hidden focus:bg-stone-100 dark:text-neutral-500 dark:focus:bg-neutral-700" aria-haspopup="menu" aria-expanded="false" aria-label="Dropdown">
-                Admissions
-                <svg class="shrink-0 size-3.5" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="m7 15 5 5 5-5"/><path d="m7 9 5-5 5 5"/></svg>
-              </button>
+            <th scope="col">
+              <!-- Sort Dropdown -->
+              <div class="hs-dropdown relative inline-flex w-full cursor-pointer">
+                <button id="hs-pro-eptsas" type="button"
```

HTMLHint (1.5.0)

[error] The id value [ hs-pro-eptsas ] must be unique.

(id-unique)